src/sentry/backup/comparators.py

```diff
@@ -418,23 +418,9 @@ def compare(self, on: InstanceID, left: JSONData, right: JSONData) -> list[Compa
                 )
             )
 
-        # Old user, password must remain constant.
+        # Old user, all fields must remain constant.
         if not right["fields"].get("is_unclaimed"):
             findings.extend(super().compare(on, left, right))
-
```

Comment:
I realized the call to `super().compare(...)` above checks all fields for equality, so the check I added in https://github.com/getsentry/sentry/pull/68223 is actually unnecessary.